src/library/dao/savedqueriesdao.h

```diff
@@ -0,0 +1,97 @@
+#ifndef SAVEDQUERIESDAO_H
+#define SAVEDQUERIESDAO_H
+
+#include <QList>
+#include <QSqlDatabase>
+#include <QString>
+
+#include "library/dao/dao.h"
+#include "util/dbid.h"
+
+#define SAVEDQUERYTABLE "savedQueries"
+
+enum SavedQueryColumns {
```

Comment:
In this case, the `SavedQueryColumns` is stored in the database as an `int` and refers also to columns indexes. It's assumed to have a direct `int` conversion and changing it co `enum class` implies adding a lot of lines like this:

```
SavedQueryColumns col = static_cast<SavedQueryColumns>(index.column());
```

Every time this enum is used the int conversion is used too. See [src/dialog/savedqueries/savedqueriestablemodel.cpp](https://github.com/daschuer/mixxx/blob/jmigual-library-redesign/src/dialog/savedqueries/savedqueriestablemodel.cpp#L28)